tests/test_editor.py

```diff
@@ -71,7 +71,7 @@ def test_rotate(self):
     def test_blur(self):
         """Tests editor blur"""
         canvas = Canvas((100, 100), color="black")
-        editor = Editor(canvas).blur(mode="gaussian", amount=10)
+        editor = Editor(canvas).blur(mode="gussian", amount=10)
```

Comment:
Gaussian is actually the correct spelling [ref](https://en.wikipedia.org/wiki/Gaussian_blur)

Reply:
there was  type error as official pil interface gives two mode only `box & gussian`

Reply:
they have restricted literals for above two only